guides/source/active_record_migrations.md

```diff
@@ -623,6 +623,26 @@ NOTE: Active Record only supports single column foreign keys. `execute` and
 `structure.sql` are required to use composite foreign keys. See
 [Schema Dumping and You](#schema-dumping-and-you).
 
+### Composite Primary Keys
+
+Sometimes a single column's value isn't enough to uniquely identify every row of a table, but a combination of two or more columns *does* uniquely identify it. This can be the case when using a legacy database schema without a single `id` column as a primary key, or when altering schemas for sharding or multitenancy.
```

Comment:
New docs should wrap to 80 characters. I haven't been pushing for that on older docs that are updated. but any new ones should follow that convention.

Reply:
I've just re-wrapped it.